src/test/java/racingCar/racingCarTest.java

```diff
@@ -5,10 +5,78 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 public class racingCarTest {
 
+    @Test
+    @DisplayName("가장 많이 전진한 자동차 이름을 반환한다.")
+    public void 가장_많이_전진한_자동차_반환(){
+        List<Car> carList = new ArrayList<>();
+        carList.add(new Car("pobi", 5));
+        carList.add(new Car("crong", 3));
+        carList.add(new Car("honux", 5));
+
+        NamedCars cars = new NamedCars(carList);
+
+        List<String> winners = WinnerCar.returnWinner(cars);
+        assertThat(winners).hasSize(2);
+        assertThat(winners).containsExactly("pobi", "honux");
+    }
+
+    @Test
+    @DisplayName("입력한 자동차 이름과 시도 횟수 만큼 자동차를 움직인 결과를 반환한다.")
+    public void 이름있는_자동차_경주_결과_반환(){
+        NamedCars cars = RacingCar.movingResultNamedCar(new NamedCars("pobi,crong,honux"), 5);
+
+        assertThat(cars.getCarList()).hasSize(3);
+
+        for(int i=0; i<3; i++) {
+            assertThat(cars.forwardCountOf(i)).isBetween(0, 5);
+        }
+
+    }
+
+    @Test
+    @DisplayName("입력한 여러대의 자동차 이름을 콤마 구분자로 구분하여 자동차 이름을 부여하고 0번 움직인 결과를 반환한다.")
+    public void 자동차_경주_초기_설정(){
+        String carNames = "pobi,crong,honux";
+
+        NamedCars cars = new NamedCars(carNames);
+
+        assertAll(
+                () -> assertThat(cars.carNameOf(0)).isEqualTo("pobi"),
```

Comment:
값을 꺼내와서 비교하려고하는 부분들이 많아보여요.
`equals`,`hashcode`함수를 재정의해서 객체간의 비교해보면 어떨까요?

```suggestion
                () -> assertThat(cars.find(1)).isEqualTo(new Car("pobi",1)),
                
                또는
                
                () -> assertThat(cars.contains(new Car("pobi",1))).isTrue(),
```

예를 들면 이런식으로요!